src/client/game.cpp

```diff
@@ -1347,6 +1349,8 @@ bool Game::createClient(const GameStartData &start_data)
 	/* Skybox
 	 */
 	sky = new Sky(-1, m_rendering_engine, texture_src, shader_src);
+	if (g_settings->getBool("enable_dynamic_shadows"))
```

Comment:
i don't understand why it's done there, just do it directly in the sky constructor, it's better to understand how sky is inited

Reply:
I don´t see wrong to set up sky that way, it depends on global shadows setting and to move it inside the sky we must add the g_settings include in sky. But if you have strong feelings on this, I can change it :)

Reply:
Perhaps at least pass the value to the Sky(...) constructor...? I.e. add a tilt parameter to the constructor and pass the value in. That way Sky does not need to include g_settings, and it'll be harder to forget to set it.

Reply:
Sky already uses g_settings... I'll move this there in a commit.